mitodl/rapid-response-xblock
# Add database models to store runs

#### What are the relevant tickets?
Part of #29 

#### What's this PR do?
Creates a `RapidResponseRun` to store information about multiple rounds of submissions. This replaces `RapidResponseBlockStatus` since this information is already in `RapidResponseRun`. 

#### How should this be manually tested?
The only user facing thing you should notice is that when you close and open a problem, the old results should go away. They will still come through the API and be present in the database but they are filtered out in the front end (for now).

#### Any background context you want to provide?
You should probably get rid of all `RapidResponseSubmission` to avoid stale data. There is no data migration from the old format into the new one
